src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -536,4 +538,13 @@
      */
     List<BannerPatternShape> getBannerPatternShapes();
 
+    /**
+     * Gets a {@link DisplaySlot} which displays only for teams
+     * with the provided color.
+     *
+     * @param color The color for the display slot
+     * @return The {@link DisplaySlot} with the provided color, or Optional.absent() if not found
+     */
+    Optional<DisplaySlot> getDisplaySlotForColor(TextColor color);
+
```

Comment:
You should also add the other methods, like `getCriteria()` and `getDisplaySlots()` returning all available ones or one by name, like with the other enum-like classes in the `GameRegistry`